params/config.go

```diff
@@ -128,6 +130,7 @@ var (
 		ByzantiumBlock:      big.NewInt(1_035_301),
 		ConstantinopleBlock: big.NewInt(3_660_663),
 		PetersburgBlock:     big.NewInt(4_321_234),
+		RamanujanBlock:      big.NewInt(0),
```

Comment:
This should probably left as nil and not zero (which would mean the this fork is activated straight away)